components/Tabs/Tabs.js

```diff
@@ -2,61 +2,67 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
     
+    this.element = element;
     // Get the custom data attribute on the Link
-    // this.data;
+    this.data = document.querySelector(`.tabs-link[data-tab='${element.dataset.tab}']`);
     
-    // Using the custom data attribute get the associated Item element
-    // this.itemElement;
+  
     
+    // Using the custom data attribute get the associated Item element
+    // ;
+     this.itemElement = document.querySelector(`.tabs-item[data-tab='${element.dataset.tab}']`);
```

Comment:
because you reference the data on the element specifically again this works, but the instructions were asking for you to already store the data value in this.data, so you would pass that in to this interpolated string instead of element.dataset.tab